wp-content/themes/pub/wporg-learn-2020/css/components/_section-heading.scss

```diff
@@ -31,7 +36,24 @@
 		}
 	}
 
+	.button.button-large {
```

Comment:
Since these styles are used in both the header and at the bottom of the page I'm wondering if we should add a new button style for this. Maybe a `components/_button.scss` file with a `button-xlarge` modifier class, I think these minimal styles will be enough:

```
.button {
	&.button-secondary {
		&.button-xlarge {
			padding-top: 12px;
			padding-bottom: 12px;
			height: auto;
			font-weight: 600;
			border-radius: 4px;
		}
	}
}
```